packages/sdk/src/utils/constants.ts

```diff
@@ -23,4 +23,6 @@ export const NON_NULL_ADDRESS = '0x0000000000000000000000000000000000000001'
 
 export const NULL_BYTES32 = '0x0000000000000000000000000000000000000000000000000000000000000000'
 
-export const MINT_GAS_LIMIT_MULTIPLIER = 1.2
+export const MINT_GAS_LIMIT_MULTIPLIER = 1.3
+
+export const MINT_FALLBACK_GAS_LIMIT = 220_000
```

Comment:
```suggestion
export const MINT_GAS_LIMIT_MULTIPLIER = 1.2

export const MINT_FALLBACK_GAS_LIMIT = 200_000
```

200_000 is what we are reliably running in our prod app w/ mints being 100k to 125k on upper end

20% should be sufficient as it's really 1 extra storage write (mostly warm) that can be missed by the estimate